### DISCUSSION TITLE
release/feature plan

### DISCUSSION THREAD

**Discussion by Druid33** [amborle/featmap#48] [Q&A]
Hi, thanks for publish code of this product.

I try use it on our new project and see some space for improvement and new features. Do you have same plan of next releases and new features?.....maybe some public link to featmap story map :smile:

**Comment by amborle** [comment]
Hi Peter! Sorry, no story map I am comfortable to share 😉

If you feel you have some new use cases you would want to see covered please post them here: https://github.com/amborle/featmap/discussions/categories/ideas

Cheers!
